bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBinding.java

```diff
@@ -108,7 +108,7 @@ public void removeKNXTypeMapper(KNXTypeMapper typeMapper) {
     @Override
     protected void internalReceiveCommand(String itemName, Command command) {
         logger.trace("Received command (item='{}', command='{}')", itemName, command.toString());
-        if (!isEcho(itemName, command)) {
+        if (!KNXConnection.getCompatibilityOH2() && !isEcho(itemName, command)) {
```

Comment:
The different behavior between and OH1 and OH2 compat1x is:

sending a _Command_ (sendCommand from Rule for example) ->
in OH1:
*internalReceiveCommand is called* (no matter if value is changed or not)

in OH2 compat1x:
*internalReceiveCommand is called AND directly after internalReceiveUpdate is called*
  (sent, no matter if value is changed or not)


sending an _Update_ (postUpdate) ->
in OH1:
*internalReceiveUpdate is called* (sent, no matter if value is changed or not)

in OH2 compat1x:
*internalReceiveUpdate is called* (sent, no matter if value is changed or not)

So running KNX1 bundle in OH2 causes two KNX-bus events. *KNXConnection.getCompatibilityOH2()* simply switches off the first of the two Events.


@watou, the Documentation of *List\<String\>* says:
> Removes the first occurrence of the specified element from this list, if it is present (optional operation). If this list does not contain the element, it is unchanged.

I believe there is not the risk of a memory leak at this point. Am I wrong?
It saves nothing if I check before removing whether the item is in the list, on the contrary.

`isEcho` has to delete a value from ignoreEventList first time it occurs, this is intended.